rudder-jetty/SPECS/rudder-jetty.spec

```diff
@@ -88,10 +88,13 @@ Requires: jre >= 1.8
 Requires: java-1.8.0-openjdk-headless
 %endif
 
+%if 0%{?suse_version} && 0%{?suse_version} == 1315 && 0%{?sle_version} && 0%{?sle_version} >= 120100
```

Comment:
It is consistent with what is documented there: https://en.opensuse.org/openSUSE:Build_Service_cross_distribution_howto